Sources/ForageSDK/Component/ForagePINTextField/ForagePINTextField.swift

```diff
@@ -105,6 +105,34 @@ public class ForagePINTextField: UIView, Identifiable, ForageElement {
         set { textField.font = newValue }
     }
     
+    /// InputWidth for the text field
+    @IBInspectable public var inputWidth: CGFloat {
+        get { return textField.inputWidth }
+        set {
+            container.alignment = .center
+            textField.inputWidth = newValue
+        }
+    }
+    
+    /// InputHeight for the text field
+    @IBInspectable public var inputHeight: CGFloat {
+        get { return textField.inputHeight }
+        set {
+            container.alignment = .center
+            textField.inputHeight = newValue
+        }
+    }
+    
+    /// ElementHeight for the text field
+    private var _elementHeight: CGFloat = 0
```

Comment:
For each of these new members' comments (both PIN + PAN comments), can we make the distinction between element height and inputHeight more explicit?

For example, something like:

```
before:
/// InputHeight for the text field
inputHeight...

after:
/// Height of the input field within the ForagePANTextField.
/// - SeeAlso: `elementHeight` to customize the height of the entire ForagePANTextField.
inputHeight...

--------------

before:
/// ElementHeight for the text field
elementHeight...

after:
/// Height of the entire ForagePANTextField element.
/// - SeeAlso: `inputHeight` to customize the height of the input field within the ForagePANTextField.
elementHeight...

or 

/// Container height of the ForagePANTextField element.
/// - SeeAlso: `inputHeight` to customize the height of the input field within the ForagePANTextField.
elementHeight...
```

Reply:
Updated. Thanks